input/projects/data/csharp-standard.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: C# Standard
+Contributor: Ecma International
+Web: https://www.ecma-international.org/
+---
+# C# language standard
+
+[TC49-TG2](https://www.ecma-international.org/task-groups/tc49-tg2/) writes the official C# standard. The scope is to standardize the syntax and semantics of a modern, component-based, general purpose, object oriented, and type-safe programming language called C# (pronounced C sharp).
+
+The committee Programme of work:
+
+1. Develop C# language standards.
```

Comment:
I have a question about how blunt to be here. I'm not convinced that we do (or ever will do) any of these three items in the way that I'd read them if I came to this page cold.

Really, we act as a proof-reading/editing phase for the language designed in the Roslyn repo as linked later on. I guess it may be reasonable to say we developer the C# language standards - we just don't develop the C# language itself. But perhaps we should just be up-front about it in this list item?

Reply:
I took this text from the [ECMA](https://www.ecma-international.org/task-groups/tc49-tg2/) page. I'll try a reword of this.

Reply:
Ah, that makes sense. To be honest, that's fine. I doubt that we need to worry about it *too* much, and we can always change it.